landbosse/excelio/XlsxManagerRunner.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, file_ops=None):
         """
         self.file_ops = file_ops if file_ops is not None else XlsxFileOperations()
 
-    def run_from_project_list_xlsx(self, projects_xlsx, log):
+    def run_from_project_list_xlsx(self, projects_xlsx,  enable_cost_and_scaling_modifications=True):
         """
```

Comment:
Did you mean to get rid of the logger?

Reply:
Yes, I meant to get rid of the logger. All the logging is done with print statements now, because they are more reliable for multi-process logging.